File: Javascript/Uri1037_Intervalo.js
var input = require("fs").readFileSync("stdin", "utf8");
var lines = input.split('\n');

var x = parseFloat(lines.shift());

if (x >= 0 && x <= 25.0000){
    console.log("Intervalo [0,25]");
}

else if(x > 25.0000 && x <= 50.0000){
    console.log("Intervalo (25,50]");
}

else if(x > 50.0000 && x <= 75.0000){
    console.log("Intervalo (50,75]");
}

else if(x > 75.0000 && x <= 100.0000){
    console.log("Intervalo (75,100]")
}

else{
    console.log("Fora de intervalo")
}

File: Python/Uri1037_Intervalo.py
vlr = float(input())

if vlr>=0 and vlr<=25.0000:
    print('Intervalo [25,50]')

if vlr>25.0000 and vlr<=50.0000:
    print('Intervalo (25,50]')

if vlr>50.0000 and vlr<=75.0000:
    print('Intervalo (50,75]')

if vlr>75.0000 and vlr<=100.0000:
    print('Intervalo (75,100]')

else:
    print('Fora de intervalo')